imageroot/bin/module-dump-state

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env sh
+
+#
+# Copyright (C) 2023 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+set -e
+
+# clean container files on exit
+cleanup () {
+    podman exec mariadb rm -rf /tmp/db_backup
+    podman exec asterisk rm -f /var/lib/asterisk/db/astdb.sqlite3.dump
+}
+
+trap cleanup EXIT
+
+## MariaDB
+# exec mariabackup to save database in /tmp/db_backup
+podman exec mariadb \
+    mariabackup \
+        --backup \
+        --target-dir=/tmp/db_backup \
+        --user=root \
+        --password="${MARIADB_ROOT_PASSWORD}"
+# copy db_backup in local filesystem
+podman cp mariadb:/tmp/db_backup db_backup
+
+
+## Asterisk
+
+# /var/lib/asterisk/db/astdb.sqlite3.dump
+# /var/lib/asterisk/sounds/*/custom
+# /var/lib/asterisk/sounds/nethcti
+# /var/spool/asterisk/voicemail
+# /var/spool/asterisk/monitor
+
+# dump asterisk sqlite db
+podman exec asterisk backup_astdb
+# copy sqlite db to local filesystem
+mkdir -p asterisk_backup/var/lib/asterisk/db
+podman cp asterisk:/var/lib/asterisk/db/astdb.sqlite3.dump asterisk_backup/var/lib/asterisk/db/astdb.sqlite3.dump
+
+# copy only custom sounds
+sounds_dirs=$(podman exec asterisk find /var/lib/asterisk/sounds -type d -name custom)
```

Comment:
This should be only one dir, can we use a volume for it?

Reply:
Not at the moment, a volume is mounted in `/var/lib/asterisk/sounds` already, there might be installations where multiple additional languages are installed, not knowing how many are there, I've preferred to get them all :sweat_smile:

Reply:
Let's keep it for now, but a rework is needed.